### PR TITLE
Fix Slider on_press and on_release in Windows.

### PR DESCRIPTION
**Pull request by saroad2**
On windows, pressing and releasing sliders does not cause the `on_change ` method to perform the same way the Cocoa implementation does.

Fixed that and fixed the example accordingly.

## PR Checklist:

- [X] All new features have been tested
- [X] All new features have been documented
- [X] I have read the **CONTRIBUTING.md** file
- [X] I will abide by the code of conduct

### CODE REVIEW

File: src/winforms/toga_winforms/widgets/slider.py [hunk 1]
```diff
@@ -39,12 +39,22 @@ def winforms_scroll(self, sender, event):
...
+        self.winforms_scroll(sender, event)
 
     def winforms_mouse_up(self, sender, event):
+        """
+        Since picking and releasing the slider is also a change, calling the
+            on_change method.
+        """
         if self.container and self.interface.on_release:
             self.interface.on_release(self.interface)
+        self.winforms_scroll(sender, event)
```
Comment (freakboy3742) [inline src/winforms/toga_winforms/widgets/slider.py:57]: I think this should go *before* the `on_release` handling... effectively, the sequence of events is "down, scroll, scroll, scroll, release".

**Review by freakboy3742 — CHANGES_REQUESTED** [review]
One minor fix and I think this is good to go; I almost made the change myself, but I wanted to make sure you agree with the logic.

File: src/winforms/toga_winforms/widgets/slider.py (hunk 1 above)
Comment (saroad2) [reply src/winforms/toga_winforms/widgets/slider.py]: Completely agree. Entered the new logic.

**Review by freakboy3742 — APPROVED** [review]
👍 Looks good to me!

**Pull request merged**